### PR TITLE
Request to implement source code for Newsweek page

### PR DESCRIPTION
**Pull request by luisvinicius09**
Constructing the Newsweek main page. 👷

- Uploading the assets.
- Creating the skeleton of the page;
- Adding all markups and styling.
- Polishing on the positioning of elements.
- Update the README file.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  padding: 2px 10px 2px 10px;
+  color: white;
+  background-color: red;
+  margin-right: 5px;
+}
+
+.my-flex p a {
+  color: red;
+  text-decoration: none;
+}
```
Comment (jubaan) [inline assets/css/styles.css]: - [x] Use efficient CSS selectors.
```css
/*-- Bad selectors --*/
.my-flex p a {
  color: red;
  text-decoration: none;
}

/*-- Good selectors --*/
.link {
  color: red;
  text-decoration: none;
}
```

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+.wrapper {
+  background-color: #fff;
+}
+
+.container-fluid {
+  background-color: #fff;
+  max-width: 1250px;
+  margin-top: 30px;
+  padding-top: 1px;
+}
```
Comment (jubaan) [inline assets/css/styles.css]: - [x] Keep code Dry by reusing same CSS rules
```css
/*-- Verbose code --*/
.wrapper {
  background-color: #fff;
}

.container-fluid {
  background-color: #fff;
  max-width: 1250px;
  margin-top: 30px;
  padding-top: 1px;
}

/*-- Dry code example 1 --*/
.wrapper,
.container-fluid {
  background-color: #fff;
}

.container-fluid {
  max-width: 1250px;
  margin-top: 30px;
  padding-top: 1px;
}

/*-- Dry code example 2 --*/

.container-fluid {
  max-width: 1250px;
  margin-top: 30px;
  padding-top: 1px;
}

/*-- apply this class to elements that need the background white --*/
.bg-white { 
  background-color: #fff;
}
```

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @luisvinicius09  and @Nasa28  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: This is a great solid start!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Keep code DRY, see example inline. And apply the same logic to the other rules that apply.
- [x] Use efficient CSS selectors, see example inline. And apply the same logic to the other inefficient CSS rules.
- [x] notice how the correct paths start with a `./` that marks the current position of the file within the project directory. Without that the browser won't know where the path to the asset should start from.
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">

```
- [x] Use `<article>` HTML semantic tag for each one of the artilces.

- [x] Opinion section elements don't look like the original

| **Original**| **Clone** |
|---|---|
| image | image
 |

- [x] Element goes beyond website limits and it is also creating a space to the right of the website

| **Original** | **Clone** |
|---|---|
| image | image |

- [x] Element doesn't look as original

| **Original** | **Clone** |
|---|---|
| image | image |

- [x] Tablet views of in the magazine is displaying  as mobile view.

| **Original** | **Clone** |
|---|---|
| image | image | 

- [x] Newsletter section doesn't look like original on tablet view

| **Original** | **Clone** |
|---|---|
| image | image | 

- [x] Footer is not looking like original on tablet view

| **Original** | **Clone** |
|---|---|
| image | image | 

- [x] Feature stories goes after top stories in mobile view.

| **Original** | **Clone** |
|---|---|
| image | image | 

- [x] Categories the opinion and debate don't have the same styling as original
image

- [x] Sections dont have responsive elements for mobile view
image

- [x] Newsletter section doesn't look like the original

| **Original** | **Clone** |
|---|---|
| image | image | 


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/node_modules
+.hintrc
+.stylelint.json
```
Comment (descholar-ceo) [inline .gitignore]: Remove this `.hintrc` file from gitiignore before you merge

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @luisvinicius09 and @Nasa28  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: .gitignore (hunk 3 above)
Comment (luisvinicius09) [reply .gitignore]: Done! Thanks for your time.

Comment (Nasa28) [reply .gitignore]: Thank you.

**Pull request merged**